types/try-to-catch/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for try-to-catch 3.0
+// Project: https://github.com/coderaiser/try-to-catch
+// Definitions by: Coderaiser <https://github.com/coderaiser>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
+
+declare function tryToCatch<PromiseData, FnArgs extends any[]>(
+  fn: (...args: FnArgs) => Promise<PromiseData>,
+  ...fnArgs: FnArgs
+): Promise<[Error] | [null, PromiseData]>;
+
+declare function tryToCatch<Data, FnArgs extends any[]>(
+  fn: (...args: FnArgs) => Data,
+  ...fnArgs: FnArgs
+): Promise<[Error] | [null, Data]>;
```

Comment:
I have often seen generic parameters to be named starting with an uppercase `T`, but I am not sure whether this is a strict convention ... Otherwise, LGTM :-)

Reply:
I think names is good and self explanatory.